torch/csrc/jit/mobile/promoted_prim_ops.cpp

```diff
@@ -94,7 +94,7 @@ void device(Stack& stack) {
 
 void device_with_index(Stack& stack) {
   std::string type = pop(stack).toStringRef();
-  int index = pop(stack).toInt();
+  auto index = pop(stack).toInt();
   std::string device_str = type + ":" + std::to_string(index);
```

Comment:
Not should use libfmt or perhaps char overload. Might actually be a bit cleaner with append here.

Reply:
Applied